### ISSUE TITLE
[BUG] Close, Minimize, Maximize button bugged out!

### ISSUE THREAD

**Issue by MatsCraft1** [AdamXweb/WhiteSurFirefoxThemeMacOS#98]
### Describe the problem

Hey, I'm so sad this bug happened.
I've been using your theme for half a year now, and it's great. It has a very clean, modern and nice feeling to it.

However, after I restarted my PC, this part just bugged out and won't stop (see screenshots)
I didn't change anything, I also tried reinstalling it, but that didn't fix anything.

I'd be grateful for any help, as I'd love to keep using it!

### Version

v1.5.2 2023-05-12 Latest

### OS

Windows 11

### Firefox Version

117.0

### Install Options

none

### Colour Theme

no

### Additional info

![32a074cc2d88e0e185edf659355cc116](https://github.com/AdamXweb/WhiteSurFirefoxThemeMacOS/assets/16339986/73c34878-d1bc-4365-bb09-cde4645fbd19)

### Please tick the boxes

- [X] You are using a [current](https://github.com/AdamXweb/WhiteSurFirefoxThemeMacOS/releases/) version of the theme. (check the date associated with the version)
- [X] You've checked that this [issue hasn't already been raised](https://github.com/adamxweb/WhiteSurFirefoxThemeMacOS/issues?q=is%3Aissue)

**Comment by 1-800-aesthetics** [comment]
Hello, I'm having the same trouble, so i started search in theme folder "C:\Users\User\AppData\Roaming\Mozilla\Firefox\Profiles\ebp285dx.default-release\chrome\WhiteSur\parts" and find buttons-fixes,css and i think here already have the fix but i don't know css is not for me

**Comment by AdamXweb** [comment]
@MatsCraft1 Thanks for reporting this issue, i'll look into this when I get a chance. Can confirm I've reproduced the issue on my VM.

**Comment by terran5151** [comment]
Same issue here, 
![image](https://github.com/AdamXweb/WhiteSurFirefoxThemeMacOS/assets/88588091/312e2741-e7a9-4c3e-84d5-62290d55d379)

**Comment by easonwong-de** [comment]
Fixed with:

```
/* Reverse the botton placing */

#titlebar .titlebar-buttonbox {
	flex-direction: row-reverse !important;
}

/* Remove default symbols in the button & resize & relocate */

#titlebar .titlebar-buttonbox .titlebar-button {
	list-style-image: none !important;
	padding: 2px !important;
	margin: 0 4px !important;
	background-color: transparent !important;
}
```

**Comment by AdamXweb** [comment]
> Fixed with:
> 
> 
> 
> ```
> 
> /* Reverse the botton placing */
> 
> 
> 
> #titlebar .titlebar-buttonbox {
> 
> 	flex-direction: row-reverse !important;
> 
> }
> 
> 
> 
> /* Remove default symbols in the button & resize & relocate */
> 
> 
> 
> #titlebar .titlebar-buttonbox .titlebar-button {
> 
> 	list-style-image: none !important;
> 
> 	padding: 2px !important;
> 
> 	margin: 0 4px !important;
> 
> 	background-color: transparent !important;
> 
> }
> 
> ```

@easonwong-de thanks for submitting a fix. Would you like to open a pull request?

**Comment by justsmbody** [comment]
> > Fixed with:
> > ```
> > 
> > /* Reverse the botton placing */
> > 
> > 
> > 
> > #titlebar .titlebar-buttonbox {
> > 
> > 	flex-direction: row-reverse !important;
> > 
> > }
> > 
> > 
> > 
> > /* Remove default symbols in the button & resize & relocate */
> > 
> > 
> > 
> > #titlebar .titlebar-buttonbox .titlebar-button {
> > 
> > 	list-style-image: none !important;
> > 
> > 	padding: 2px !important;
> > 
> > 	margin: 0 4px !important;
> > 
> > 	background-color: transparent !important;
> > 
> > }
> > ```
> 
> @easonwong-de thanks for submitting a fix. Would you like to open a pull request?

Could you update the theme with the bug fixed?

**Comment by easonwong-de** [comment]
I hope #101 works.

**Comment by AdamXweb** [comment]
Fixed from #101
